delta-app/src/main/java/io/cdap/delta/store/DraftService.java

```diff
@@ -473,11 +473,19 @@ private TableSummaryAssessment summarize(String db, @Nullable String schema, Str
       if (columnAssessment.getSupport() == ColumnSupport.NO) {
         numUnsupported++;
       } else if (columnAssessment.getSupport() == ColumnSupport.PARTIAL) {
-        numPartial++;
+        Set<SourceColumn> columns = table.getColumns();
+        SourceColumn existingColumn = columns.stream()
+          .filter(column -> columnAssessment.getSourceName().equals(column.getName()))
+          .findAny()
+          .orElse(null);
+
+        if (existingColumn == null || !existingColumn.isSuppressWarning()) {
```

Comment:
in which case it will be null and the partial num should increase ?

Reply:
column selection is not required, so the array of column selection can be empty, in which case the column is supposed to be replicated

Reply:
got it. LGTM